app/src/main/java/com/github/studydistractor/sdp/data/Event.kt

```diff
@@ -47,10 +48,11 @@ data class FirebaseEvent(
     val start: String? = null,
     val end: String? = null,
     val lateParticipation: Boolean = false,
+    val numberOfPoints: Int = 0,
     val chatId: String? = null) {
     fun toEvent(): Event {
         return Event(
-            eventId, name!!, description!!, lat!!, long!!, start!!, end!!, lateParticipation, chatId!!
+            eventId, name!!, description!!, lat!!, long!!, start!!, end!!, lateParticipation, numberOfPoints, chatId!!

```

Comment:
You should add some the !! operator after the `numberOfPoints` as the others fields to ensure data integrity